app/models/report.rb

```diff
@@ -1548,6 +1548,55 @@ def self.report_top_uploads(report)
     end
   end
 
+  def self.report_top_ignored_users(report)
+    report.modes = [:table]
+
+    report.labels = [
+      {
+        type: :user,
+        properties: {
+          id: :ignored_user_id,
+          username: :ignored_username,
+          avatar: :ignored_user_avatar_template,
+        },
+        title: I18n.t("reports.top_ignored_users.labels.ignored_user")
+      },
+      {
+        type: :number,
+        properties: [
+          :ignores_count,
+        ],
+        title: I18n.t("reports.top_ignored_users.labels.ignores_count")
+      }
+    ]
+
+    report.data = []
+
+    sql = <<~SQL
+    SELECT
```

Comment:
This would look better if the SQL query was indented ▶️

Reply:
Agreed, and this comment is the most trivial thing I can ever think of bringing up, but `as` should be capitalised since those are SQL keywords we generally capitalise all of them.

Reply:
I think so 👍  I am unsure why do we have the indentation in all the other report methods this same way though.